scripts/superdesk-search/react.js

```diff
@@ -1019,7 +1019,8 @@
 
                         takekey: function(props) {
                             if (props.item.anpa_take_key) {
-                                return React.createElement('div', {className: 'takekey', key: 'takekey'}, props.item.anpa_take_key);
+                                return React.createElement('div', {className: 'takekey', key: 'takekey'},
+                                    gettextCatalog.getString(props.item.anpa_take_key));
```

Comment:
just wondering - when you use `gettextCatalog` instead of `gettext` - is it to only translate but avoid another entry in po files?